webpack.config.js

```diff
@@ -102,73 +110,33 @@ const config = {
         { from: 'locales', to: 'locales' },
       ],
     }),
+    new GenerateSW({
```

Comment:
A service worker is also generated in development mode. But in `ServiceWorkerManager` the code still doesn't install it, so there's no change for the users. But it's now easier to trigger if we need to debug things. It's also easier to inspect the generated code in development mode where it's not minified.

Reply:
I don't think this slows down the dev server build time much... but your question reminded me something else: this may generate a warning when in watch mode, which would be annoying. I'll double-check this.

Reply:
as I remembered:
![Screen Shot 2022-07-29 at 19 09 03](https://user-images.githubusercontent.com/454175/181809796-366f7d7c-b749-4111-9f3e-97f1bad2d0bf.png)

Therefore I'll add it in production only after all...